src/app/components/Curation/CurationPromo/index.tsx

```diff
@@ -48,16 +48,17 @@ const CurationPromo = ({
     (type === 'photogallery' && `${photoGalleryTranslation}, `);
 
   return (
-    <Promo>
+    <Promo css={styles.promo} className="">
       {imageUrl && (
         <Promo.Image
           src={imageUrl}
           alt={imageAlt}
           lazyLoad={lazy}
           isAmp={isAmp}
+          {...(isLite && { css: styles.image })}
```

Comment:
Did have the lite check in the styles but interfered with styles set in the parent using some nth child business https://github.com/bbc/simorgh/blob/8b1fc0ae4aff4ef0e9ac6e5cc344cc2fe9f5bbd1/src/app/components/Curation/CurationGrid/index.styles.ts#L28, which caused the media icons to shrink, so had to put the check in here so no css attribute is applied.